designer/client/src/actions/nk/displayProcessCounts.ts

```diff
@@ -15,11 +16,34 @@ export function displayProcessCounts(processCounts: ProcessCounts): DisplayProce
     };
 }
 
+const checkActualCounts = (processCounts: ProcessCounts, processToDisplay: Process) => {
+    const processCountsName = Object.keys(processCounts).sort((a, b) => a.localeCompare(b));
+    const nodes = processToDisplay.nodes
```

Comment:
what about rename it from`nodes` to `uncountableNodes`?

Reply:
okay :)